src/python/tensorflow_cloud/core/tests/integration/run_on_script_test.py

```diff
@@ -21,18 +21,18 @@
 import tensorflow_cloud as tfc
 
 # Following are the env variables available in test infrastructure:
-#
+
 # The staging bucket to use for cloudbuild as well as save the model and data.
-# TEST_BUCKET = os.environ['TEST_BUCKET']
-#
+_TEST_BUCKET = os.environ["TEST_BUCKET"]
```

Comment:
This ought to be 'GCS_TEST_BUCKET', per CI configuration

Reply:
It is exported as TEST_BUCKET in the CI config.